### ISSUE TITLE
Show footer with page height adjusted

### ISSUE THREAD

**Issue by lpugin** [rism-ch/verovio#1802]
Is there a way when `--adjust-page-height` is given as an option, for the page footer to **be** displayed? (as if we just cut the white space out of the score with scissors and taped the footer back onto the score).

I think that should be the default behavior of `--adjust-page-height`, (with `--footer none` required if you really want to hide the footer).

_Originally posted by @earboxer in https://github.com/rism-ch/verovio/issues/736#issuecomment-721966406 and commented by @craigsapp in https://github.com/rism-ch/verovio/issues/736#issuecomment-722032838_

**Comment by earboxer** [comment]
> Maybe a musicHeight could be used instead of pageMarginBottom? And/or the footerHeight would be incorporated into the pageMarginBottom when calculating the musicHeight.

I think `pageMarginBottom` would be a more meaningful option for configuration than `musicHeight`.
People don't want the footer overlapping the margin, right?
I think we would calculate musicHeight as follows:

```
footerHeight = currentPageHasFooter() ? some_static_value : 0
musicHeight = pageHeight - pageMarginTop - pageMarginBottom - footerHeight
```

Is it okay for `footerHeight` to be either `0` or `some_static_value`? In [my comment](https://github.com/rism-ch/verovio/issues/736#issuecomment-721989295), that value would be used to calculate the negative y offset.

**Comment by lpugin** [comment]
Implemented in #1835
